extensions/who-is-visiting-this-content/app.R

```diff
@@ -152,20 +280,29 @@ server <- function(input, output, session) {
       ungroup() |>
 
       # Join to usernames
-      left_join(user_names(), by = "user_guid") |>
-      replace_na(list(full_name = "[Anonymous]")) |>
+      left_join(users(), by = "user_guid") |>
+      replace_na(list(display_name = "[Anonymous]")) |>
       arrange(desc(timestamp)) |>
-      select(timestamp, full_name, username)
+      select(user_guid, display_name, timestamp)
+
+    # Conditionally filter by selection from other table
+    filter_selection <- getReactableState("aggregated_visits", "selected")
+    if (isTruthy(filter_selection)) {
+      filter_guid <- aggregated_visits_data()[filter_selection, ] |>
+        pull(user_guid)
+      all_visits |>
+        filter(if (is.na(filter_guid)) is.na(user_guid) else user_guid == filter_guid)
```

Comment:
Do we need this `if` clause here? When is `filter_guid` `NA`? Is it for Anonymous visits only?

Separately: Are you sure that `user_guid == filter_guid` is doing what you want here when there is more than one GUID? I would have expected `user_guid %in% filter_guid`

Reply:
The code that was up when you reviewed was before I had implemented the multiple filter behavior. Now this code reads:

```r
    if (length(input$selected_users) > 0) {
      all_visits <- filter(all_visits, user_guid %in% input$selected_users)
    }
    all_visits
```

Reply:
Yes, that is definitely clearer that way for sure